common/config/.golangci.yml

```diff
@@ -53,6 +53,7 @@ linters:
   - unparam
   - unused
   - gci
+  - gosec
```

Comment:
This gets copied in from https://github.com/istio/common-files. So it will be reverted. What I tend to do for changes like this is commit it to istio/istio (like you are doing) so that it gets tested in CI, but then immediately after send a change for the same file in https://github.com/istio/common-files so that it doesn't get reverted on the next update.

Reply:
Got it, will do that after this is merged

Reply:
One thing to note is that when it's merged in common-files it will get copied out to the other Istio repos. If you think the linter will fail in another repo, it would be could to make the changes there before getting things merged in common-files.